### PR TITLE
[EventHubs] 5.11.6 release prep

### PR DESCRIPTION
**Pull request by swathipil**

### CODE REVIEW

**Review by kashifkhan — APPROVED** [review]

File: sdk/eventhub/azure-eventhub/CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,14 @@
 
 ## 5.11.6 (2024-02-12)
 
+This version and all future versions will require Python 3.8+. Python 3.7 is no longer supported.
+
 ### Features Added
 - Added `keep_alive` functionality on EventHubProducerClient to allow for long-living producers. [#33726](https://github.com/Azure/azure-sdk-for-python/issues/33726)
 
+### Other Changes
```
Comment (l0lawrence) [inline sdk/eventhub/azure-eventhub/CHANGELOG.md:10]: Do we need other changes?

**Review by l0lawrence — APPROVED** [review]

File: sdk/eventhub/azure-eventhub/CHANGELOG.md (hunk 1 above)
Comment (swathipil) [reply sdk/eventhub/azure-eventhub/CHANGELOG.md:10]: yes to notify that we're adding 3.12

**Pull request merged**